File: Android_Networking/PhpPre001/shopping/models/product.php
<?php
class Product
{
    // Columns
    private $id;
    private $name;
    private $price;
    private $created;
    private $category_id;
    private $image_url;
    function __construct($id, $name, $price, $created, $category_id, $image_url)
    {
        $this->id = $id;
        $this->name = $name;
        $this->price = $price;
        $this->created = $created;
        $this->category_id = $category_id;
        $this->image_url = $image_url;
    }
    function getName()
    {
        return $this->name;
    }
    function getId()
    {
        return $this->id;
    }
    function getPrice()
    {
        return $this->price;
    }
    function getCreated()
    {
        return $this->created;
    }
    function getCategoryId()
    {
        return $this->category_id;
    }
    function getImageUrl()
    {
        return $this->image_url;
    }
    public function toString()
    {
        return $this->name;
    }
}
?>

File: Android_Networking/Demo15306/app/src/main/java/com/example/myapplication/Model/Product.java
package com.example.myapplication.Model;

import android.os.Parcel;
import android.os.Parcelable;

public class Product {
    private Integer id, category_id;
    private Double price;
    private String product_name, image_url;

    public Product() {
    }

    public Product(Integer id, Integer category_id, Double price, String product_name, String image_url) {
        this.id = id;
        this.category_id = category_id;
        this.price = price;
        this.product_name = product_name;
        this.image_url = image_url;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Integer getCategory_id() {
        return category_id;
    }

    public void setCategory_id(Integer category_id) {
        this.category_id = category_id;
    }

    public Double getPrice() {
        return price;
    }

    public void setPrice(Double price) {
        this.price = price;
    }

    public String getProduct_name() {
        return product_name;
    }

    public void setProduct_name(String product_name) {
        this.product_name = product_name;
    }

    public String getImage_url() {
        return image_url;
    }

    public void setImage_url(String image_url) {
        this.image_url = image_url;
    }

}
